.travis/Dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR /repos
 
 COPY . /repos/src/neonavigation
 RUN cd /repos/src \
-  && git clone --depth=1 https://github.com/at-wat/neonavigation_msgs.git \
+  && git clone --branch=safety-limiter-msgs --depth=1 https://github.com/at-wat/neonavigation_msgs.git \
```

Comment:
Needs to be removed before merging the PR.

Reply:
The PR on neonavigation_msgs has been merged.